Makefile

```diff
@@ -103,7 +103,7 @@ test-java-e2e:
 ifdef TIMEOUT
 	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoJavaE2e" -ginkgo.v -timeout $(TIMEOUT)
 else
-	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoJavaE2e" -ginkgo.v
+	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="odoJavaE2e" -ginkgo.succinct
```

Comment:
Not very sure I understand this completely, but is `-ginkgo.succinct` also required above in line 104 and line below in 113?
The tests results look concise and great...
But otherwise +2 from me on this PR.. Also, please ensure that the PR is rebased

Reply:
fixed!